server/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java

```diff
@@ -1938,6 +1938,27 @@ public void test_bit_string_can_be_inserted_and_queried() throws Exception {
         );
     }
 
+    @Test
+    @UseRandomizedSchema(random = false)
+    @UseJdbc(0) // to avoid flakiness, otherwise response can be wrapped into B'' or not.
+    public void test_bit_string_operators_in_where_clause() throws Exception {
+        execute("CREATE TABLE employees (name text, status BIT(3), comment TEXT)");
+        execute("INSERT INTO employees VALUES ('Anna', B'100','Anna works remote')");
+        execute("refresh table employees");
+
+        execute("SELECT * FROM employees WHERE status & B'100' = B'100'");
```

Comment:
We can't backport this as bitwise operators are master only. Afaik using a bitwise operator isn't required for the fix validation, so lets remove.